src/main/java/org/isf/stat/manager/JasperReportsManager.java

```diff
@@ -1097,4 +1099,37 @@ private static Date toDate(LocalDate localDate) {
 		return Optional.ofNullable(localDate).map(t -> Date.from(t.atStartOfDay().atZone(ZoneId.systemDefault()).toInstant())).orElse(null);
 	}
 
-}
+	public JasperReportResultDto getInventoryReportPdf(MedicalInventory inventory, String jasperFileName, int printQtyReal) throws OHServiceException {
+		try {
+			Map<String, Object> parameters = new HashMap<>();
+			Hospital hosp = hospitalManager.getHospital();
+			String status = MessageBundle.getMessage(inventory.getStatus());
```

Comment:
This cannot work, the `inventory.getStatus()` is not a bundle key but just a string from `enum InventoryStatus`.
If you want to have its translation I think we have to add some method in the `MedicalInventoryManager`.
I think that in the same way we could also add the `enum InventoryType`

```suggestion
			String status = MessageBundle.getMessage(inventory.getStatus());
```